src/luger.erl

```diff
@@ -195,10 +195,11 @@ throttle_channel(Channel, Now, _, _) ->
     ets:insert(?TABLE, {Channel, 1, Now}).
 
 
-record_metrics(#state{statsd = true}, Priority, Channel) ->
+record_metrics(#state{optics = true}, Priority, Channel) ->
     Key = [<<"luger.">>, luger_utils:priority_to_list(Priority), $., Channel],
-    statsderl:increment(Key, 1, 1.0);
-record_metrics(#state{statsd = false}, _Priority, _Channel) ->
+    erl_optics:counter_inc_alloc(Key, 1);
+
```

Comment:
So pretty sure that until we're fully ready to move on to optics (migrated all dashboards and whatnot) we'll need to continue sending to both statsd and optics so I think you'll need support for both.